package.json

```diff
@@ -36,7 +39,7 @@
     "test-dev": "playwright test --debug",
     "storybook": "storybook dev -p 6006",
     "storybook-docs": "storybook dev --docs --no-manager-cache",
-    "build-storybook": "storybook build --docs -o prime"
+    "build-storybook": "storybook build --docs -o prime && touch prime/.nojekyll"
```

Comment:
I realized recently that this likely isn't necessary since you can instruct the github action to clear the `gh-pages` directory _except_ the `.nojekyll` file, so only one commit when making the repo is necessary

Reply:
My intended solution is to replace the `gh-pages` branch-based deploy with [actions/upload-pages-artifact](https://github.com/actions/upload-pages-artifact) and [actions/deploy-pages](https://github.com/actions/deploy-pages), which skips Jekyll entirely and avoids the need for the file.

I've got this set up on [my personal site](https://github.com/mcous/mcous.github.io/blob/05dbf38aa7cd15b948a95f3260220b2ad7b5e130/.github/workflows/ci.yaml#L29-L50), if you're curious!